packages/airnode-utilities/src/logging.ts

```diff
@@ -0,0 +1,36 @@
+import { TextEncoder } from 'util';
```

Comment:
This is the new approach.
This is just for CLI utilities. I'm separately investigating/implementing Pino for Airnode's main logging facilities.

Reply:
Why using the TextEncoder instead of plain old console.log?

Reply:
Removed :)